source/sync/data-model/update-schema.txt

```diff
@@ -178,6 +178,13 @@ required property to both the client model and the Atlas schema.
 However, you should consider making the property optional to avoid the need to 
 modify existing Atlas documents. 
 
+.. note:: Required Properties Missing in a Schema Subset Will Default to Zero for New Documents
```

Comment:
Sorry I think I misunderstood your question on slack about this - we actually do this every time for data coming from Realm. this section LGTM if we just remove the references to "new"